generators/app/index.js

```diff
@@ -418,7 +418,7 @@ module.exports = class extends BaseGenerator {
         if (!this.skipClient) {
           if (this.clientFramework === 'angularX') {
             // add dependency required for displaying diffs
-            this.addNpmDependency('ng-diff-match-patch', '2.0.6');
+            this.addNpmDependency('ng-diff-match-patch', '3.0.1');
```

Comment:
Where does this version comes from? https://github.com/elliotforbes/ng-diff-match-patch only contains 2.0.6 version

Reply:
Sorry for the late response, I didn't receive notification. How do you want to me to do that (revert only this changes) ? To be honest, it's my first pull request :)

Reply:
This version comes directly from [NPM](https://www.npmjs.com/package/ng-diff-match-patch/v/3.0.1). An issue where opened to knows where come from this version 3.0.1 [here](
https://github.com/elliotforbes/ng-diff-match-patch/issues/39 )

Reply:
OK. I'd rather wait to know the origin of this npm release.